lib/TokenManager.ts

```diff
@@ -201,10 +246,20 @@ export class TokenManager {
   on: (event: string, handler: Function, context?: object) => void;
   off: (event: string, handler: Function) => void;
   hasExpired: (token: Token) => boolean;
+  
+  // This is exposed so we can get storage key agnostic tokens set in internal state managers
+  _getTokens: () => Promise<Tokens>;
```

Comment:
Why are we hiding this with "_" prefix?

Reply:
The app developers know the keys they defined, they can keep using `tokenManager.get` with no problems. This method is only for internal state managers to get tokens without knowing the keys. The idea to hide it is to expose as little APIs as possible to give us less maintenance burden.

Reply:
I find many times when I want to get (or set) the tokens without caring too much about the keys. One very common place for setting all tokens is with the result resolved from `parseFromUrl` or `handleAuthentication`, or widget "success" callback. It's a token hash and I just want to put them in the TokenManager. so `setTokens` would be very useful. It makes sense to provide the reverse so `setTokens` and `getTokens` can be new methods. I think there are good reasons to encourage this way if we are going to add extra token types